SchedulerJobs/SchedulerJobs/Dockerfile

```diff
@@ -3,17 +3,17 @@ WORKDIR /version
 COPY .git .git
 RUN /tools/dotnet-gitversion /ensureassemblyinfo /updateassemblyinfo AssemblyInfo.cs /nofetch /config /GitVersion.yml
 
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS installer-env
-RUN mkdir -p /home/site/wwwroot
-COPY SchedulerJobs .
-COPY --from=gitversion /version/AssemblyInfo.cs .
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-buster-slim AS base
```

Comment:
the changes in this file are not right.

you should only need to change the lines that install the packages (lines 1, 6 and 13) to install .NET 6 instead of 3/3.1.

Reply:
thanks did the changes